src/legacy/core_plugins/kbn_vislib_vis_types/public/kbn_vislib_vis_types.js

```diff
@@ -28,11 +29,18 @@ import horizontalBarVisTypeProvider from './horizontal_bar';
 import gaugeVisTypeProvider from './gauge';
 import goalVisTypeProvider from './goal';
 
-VisTypesRegistryProvider.register(histogramVisTypeProvider);
-VisTypesRegistryProvider.register(lineVisTypeProvider);
-VisTypesRegistryProvider.register(pieVisTypeProvider);
-VisTypesRegistryProvider.register(areaVisTypeProvider);
-VisTypesRegistryProvider.register(heatmapVisTypeProvider);
-VisTypesRegistryProvider.register(horizontalBarVisTypeProvider);
-VisTypesRegistryProvider.register(gaugeVisTypeProvider);
-VisTypesRegistryProvider.register(goalVisTypeProvider);
+async function registerItems() {
+  const $injector = await chrome.dangerouslyGetActiveInjector();
```

Comment:
@ppisljar `chrome.dangerouslyGetActiveInjector();` doesn't return in the browser tests and some 80 tests fail. Perhaps we need an alternative way of registering these - what the best way to do it for a karma test?

Reply:
hmm ... interesting .... i had a feeling we did this in other places that were tested by karma and it was working, will try to look into it later today if time allows.